cookbook-react/src/recipes/collapses/collapse-cmpc/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode, useState } from 'react';
+import cn from 'classnames';
+
+import styles from './styles.module.scss';
+
+interface CollapsibleProps {
+  children?: ReactNode;
+  title?: string;
+  className?: string;
+  isCollapsed?: boolean;
+  actions?: ReactNode;
+}
+
+function Collapsible({ title, className, isCollapsed, children, actions }: CollapsibleProps) {
+  const [collapsed, setIsCollapsed] = useState(isCollapsed);
+
+  const handleCollapse = () => {
+    setIsCollapsed(!collapsed);
+  };
+
+  return (
+    <div className={`${className ? className : ''} ${styles.collapsibleContainer} column`}>
+      <div className="row space-between" onClick={handleCollapse}>
+        {title && <span className={styles.title}>{title}</span>}
+        <div className="row middle">
+          {collapsed && actions}
+          <button type="button" className={cn(styles.icon, { [styles.iconClicked]: collapsed })} />
+        </div>
+      </div>
+      {!collapsed && <div className="m-top-10">{children}</div>}
+    </div>
+  );
+}
+
+Collapsible.defaultProps = {
```

Comment:
Add a comment above this clarifying that this should be erased when using it in a real project, it's just as an example